timelock-corruption-detection/src/test/java/com/palantir/timelock/corruption/detection/TimeLockCorruptionDetectionHelper.java

```diff
@@ -105,6 +105,31 @@ void assertViolationDetectedForNamespaceAndUseCases(
         return timeLockCorruptionTestSetup.createStatLogForNamespaceAndUseCase(namespaceAndUseCase);
     }
 
+    void assertClockWentBackwards() {
```

Comment:
Maybe rename to `assertClockWentBackwardsInNextBatch` to be clearer? Same for the method below